nss-day-cohort-29/welcome-to-nashville-cyclones
# All fields are searchable by feature, food, meetup, and event/concert

# Description

All fields are searchable by feature, food, meetup, and event/concert


## Type of change

Please delete options that are not relevant.





# Testing Instructions for Change Made

1. `git fetch --all`
2. `git checkout jr-result-filter`
3. run grunt
4. run http-server
5.verify that all fields are searchable by feature, food, meetup, and event/concert


# Checklist:
